css/_base.scss

```diff
@@ -1,28 +1,40 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    
+    color: $off-black;
+    font-family: $font-body;
+}
```

Comment:
I was having problems with default margin/paddings, boilerplate to remove those.  
Global color/font-family moved in here -> applies to all elements by default.